docs/contributing/development-setup/codespaces.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Github Codespaces'
+description: ''
+---
+
+GitHub Codespaces is a cloud development platform that enables developers to write, run, and debug code directly in their browser, seamlessly integrated with GitHub.
+
+### Steps to setup Codespaces
+
+1. Launch a Codespace by clicking "Create Codespace" in the main menu.
```

Comment:
1.Go to [Activepieces' repo](https://github.com/activepieces/activepieces).
2.Click Code <>, then under codespaces click create codespace on main.